### PR TITLE
Export environment variables & clean up some /photo stuff

### PR DESCRIPTION
**Pull request by skalnik**
/cc @sshirokov

### CODE REVIEW

File: server/app.rb [hunk 1]
```diff
@@ -9,11 +9,12 @@
 
 module MakeMe
   class App < Sinatra::Base
-    PID_FILE  = File.join('tmp', 'make.pid')
-    LOG_FILE  = File.join('tmp', 'make.log')
-    FETCH_MODEL_FILE = File.join('data', 'fetch.stl')
-    CURRENT_MODEL_FILE = File.join('data', 'print.stl')
-    GRUE_CONFIG = File.join('config', 'grue-make-me.config')
+    RACK_ROOT          = ENV['RACK_ROOT'] || File.expand_path('.')
```
Comment (sshirokov) [inline server/app.rb]: How about making relative to `__FILE__` instead of `pwd`

File: server/app.rb [hunk 2]
```diff
@@ -58,9 +59,9 @@ def progress
     end
 
     get '/photo' do
-      imagesnap = File.join(File.dirname(__FILE__), '..', 'vendor', 'imagesnap', 'imagesnap')
```
Comment (sshirokov) [inline server/app.rb:61]: Is what I mant [here](https://github.com/make-me/make-me/pull/75/files#r2614258)

File: server/app.rb [hunk 3]
```diff
@@ -70,8 +71,6 @@ def progress
 
       # Pick one safely and use it
       camera = cameras[params[:camera].to_i % cameras.length].strip
-      puts "Picked camera: [#{camera}]"
```
Comment (sshirokov) [inline server/app.rb:73]: :crying_cat_face:

File: server/app.rb (hunk 1 above)
Comment (skalnik) [inline server/app.rb]: That's way too reasonable.

File: server/app.rb (hunk 3 above)
Comment (skalnik) [inline server/app.rb:73]: `imagesnap` prints this out anyway…

Comment (sshirokov) [inline server/app.rb:73]: :smile_cat:

File: server/app.rb [hunk 4]
```diff
@@ -131,6 +131,8 @@ def progress
       # Slicing usually is usually under 5 seconds, so if the process runs
       # longer, it's probably printing correctly
       begin
+        p make_stl
+        p APP_ROOT
```
Comment (sshirokov) [inline server/app.rb]: :fire: :fire:

**Comment by sshirokov** [comment]
Is this DTM?

**Pull request merged**